docs/source/guides/user/chapters/configuring.rst

```diff
@@ -55,65 +55,24 @@ Avocado will read the config files present in the git repos, and will ignore
 the system wide config files. Running ``avocado config`` will let you know
 which files are actually being used.
 
-Plugin config files
--------------------
-
-There are two ways to extend settings of extra plugin configuration. Plugins
-can extend the list of files parsed by ``Settings`` object by using
-``avocado.plugins.settings`` entry-point (Python-way) or they can simply drop
-the individual config files into ``/etc/avocado/conf.d`` (linux/posix-way).
-
-`avocado.plugins.settings`
-~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-This entry-point uses ``avocado.core.plugin_interfaces.Settings``-like object
-to extend the list of parsed files. It only accepts individual files, but you
-can use something like ``glob.glob("*.conf")`` to add all config files inside a
-directory.
-
-You need to create the plugin (eg. ``my_plugin/settings.py``)::
-
-   from avocado.core.plugin_interfaces import Settings
-
-   class MyPluginSettings(Settings):
-       def adjust_settings_paths(self, paths):
-           paths.extend(glob.glob("/etc/my_plugin/conf.d/*.conf"))
-
-
-And register it in your ``setup.py`` entry-points::
-
-   from setuptools import setup
-   ...
-   setup(name="my-plugin",
-         entry_points={
-             'avocado.plugins.settings': [
-                 "my-plugin-settings = my_plugin.settings.MyPluginSettings",
-                 ],
-             ...
+Configuring via command-line
+----------------------------
 
-Which extends the list of files to be parsed by settings object. Note this
-has to be executed early in the code so try to keep the required deps
-minimal (for example the `avocado.core.settings.settings` is not yet
-available).
+Besides the configuration files, the most used features can also be configured
+by command-line arguments.  For instance, regardless what you have on your
+configuration files, you can disable sysinfo logging by running:
 
-`/etc/avocado/conf.d`
-~~~~~~~~~~~~~~~~~~~~~
+.. code-block::
 
-In order to not disturb the main Avocado config file, those plugins, if they
-wish so, may install additional config files to
-``/etc/avocado/conf.d/[pluginname].conf``, that will be parsed after the system
-wide config file. Users can override those values as well at the local config
-file level. Considering the config for the hypothethical plugin ``salad``:
+   $ avocado run --sysinfo off /bin/true
 
-.. code-block:: ini
 
-    [salad.core]
-    base = ceasar
-    dressing = ceasar
+.. note:: Please keep in mind that sysinfo option will be a boolean
+  command-line option soon.
 
-If you want, you may change ``dressing`` in your config file by simply adding a
-``[salad.core]`` new section in your local config file, and set a different
-value for ``dressing`` there.
+So, command-line options always will have the highest precedence during the
+configuration parsing. Use this if you would like to change some beahviour on
```

Comment:
There is a typo here "behavior". I guess you could quickly run the entire doc through a spell checker to detect all of these and eventually fix them in one go.

Reply:
Thanks, @pevogam will run.